README.md

```diff
@@ -6,15 +6,18 @@
 [![Documentation](https://docs.rs/rand/badge.svg)](https://docs.rs/rand)
 [![Minimum rustc version](https://img.shields.io/badge/rustc-1.22+-yellow.svg)](https://github.com/rust-lang-nursery/rand#rust-version-requirements)
 
-A Rust library for random number generators and other randomness functionality.
+A Rust library for random number generation.
 
-See also:
+Rand provides utilities to generate random numbers, to convert them to useful
+types and distributions, and some randomness-related algorithms.
 
-*   [rand_core](https://crates.io/crates/rand_core)
+Part of the functionality of Rand lives in the [rand_core](
+https://crates.io/crates/rand_core) crate, which is not intended for users but
+primarily for crates implementing RNGs.
 
-Documentation:
+API reference:
 [master branch](https://rust-lang-nursery.github.io/rand/rand/index.html),
-[by release](https://docs.rs/rand)
+[by release](https://docs.rs/rand).
```

Comment:
Considering what docs.rs is doing at the moment, it may be best to link directly to the current minor version; i.e. `docs.rs/rand/0.5` for the next release (doesn't work at the moment, but 0.4 does so should be fine for the release).